NeosModLoader/ModLoader.cs

```diff
@@ -10,7 +10,7 @@ namespace NeosModLoader
 {
 	public class ModLoader
 	{
-		internal const string VERSION_CONSTANT = "1.11.3";
+		internal const string VERSION_CONSTANT = "1.12.0";
```

Comment:
I think that version bumps should usually just be done by @zkxs directly to the main branch in a commit that also tags it as the v1.12.0 for example.

Reply:
Yeah, you're probably right. It'd definitely make the commit log easier to read. I'll start doing that in the future.